docs/marketplace/add_to_marketplace/add_item_by_type/add_custom_resource.md

```diff
@@ -0,0 +1,256 @@
+---
+id: add_custom_resource
+title: Publish your custom resources to the Marketplace
+sidebar_label: Create a custom resource
+---
+
+It is possible to include in the Marketplace Kubernetes [custom resources](/console/design-your-projects/custom-resources.md) to extend the Kubernetes API or include your own API into the project.
```

Comment:
I'd not start the sentence like this.

Something like 

> when working in the design section of a project you can define and configure custom resources, such resources may be created from the marketplace and you can add your own Custom Resources to the marketplace, this is how..

?

Reply:
I've updated this, but you have anything else in mind let me know!